runtime/gc_glue_java/MarkingDelegate.hpp

```diff
@@ -131,13 +134,14 @@ class MM_MarkingDelegate
 		return 0;
 	}
 
-	MMINLINE void doSlot(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, omrobjectptr_t *slotPtr);
+	MMINLINE void doSlot(MM_EnvironmentBase *env, omrobjectptr_t *slotPtr);
 #if JAVA_SPEC_VERSION >= 24
-	void doContinuationSlot(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, omrobjectptr_t *slotPtr, GC_ContinuationSlotIterator *continuationSlotIterator);
+	void doContinuationSlot(MM_EnvironmentBase *env, omrobjectptr_t *slotPtr, GC_ContinuationSlotIterator *continuationSlotIterator);
 #endif /* JAVA_SPEC_VERSION >= 24 */
-	void doStackSlot(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, omrobjectptr_t *slotPtr, void *walkState, const void* stackLocation);
+	void doStackSlot(MM_EnvironmentBase *env, omrobjectptr_t *slotPtr, void *walkState, const void* stackLocation);
 	void scanContinuationNativeSlots(MM_EnvironmentBase *env, omrobjectptr_t objectPtr);
 
+	void scanJ9VMContinuationSlots(MM_EnvironmentBase *env, J9VMContinuation *continuation,  omrobjectptr_t continuationObjPtr, bool stackFrameClassWalkNeeded);
```

Comment:
hm, we need a better name to clarify it has similarities with scanContinuationNativeSlots, but it's doing less (no synchronization with GC threads)
for example: scanContinuationNativeSlotsNoSync